docs/advanced-topics/legacy/tool-legacy-registration.rst

```diff
@@ -0,0 +1,495 @@
+quick registration, we look through your Quay.io images and see if any
+are set up as `automated
+builds <https://docs.quay.io/guides/building.html>`__. Using those to
+track back to your GitHub, Bitbucket, or GitLab accounts, we list all
+pairs of Docker images with git repositories that you have access to on
+those services. When we discover both of these, we create an unpublished
+entry in the interface below. We consider these tools as automated since
+new versions of the tool will be automatically added.
+
+.. figure:: /assets/images/docs/register_ui.png
+   :alt: My Tools
+
+   My Tools
+
+The left side menu is a list of all image repositories associated with
+the user, grouped lexicographically by namespace. Each tool is named
+after the docker location of the associated Docker image, in this
+example, ``quay.io/collaboratory/dockstore-tool-bamstats``. Detailed
+information and links for each tool are located on the 'Info' tab. The
+'Launch' tab includes commands for launching the tool locally with the
+Dockstore CLI. Settings such as the path to the Dockerfile and the
+Descriptor files can be modified on a per-tag basis in the 'Versions'
+tab. The Dockerfile, CWL/WDL Descriptor and test parameter files may be
+viewed in the 'Files' tab, by the Version tag (corresponding to a Git
+tag/branch). Finally, 'Manage labels' (located above the tabs) allows
+you to add/edit keywords that you want to be associated with a workflow
+for efficient searching and grouping.
+
+We also look for ``/test.cwl.json`` and ``/test.wdl.json`` in the git
+repositories on quick registration. These are the default test parameter
+file locations. Whenever a new version is added, we will check for these
+default files. You can also change these after quick registration. They
+will be applied to all versions that have not been edited, as well as
+any new versions that may appear.
+
+A tool is not visible on the public 'Tools' listing unless it is
+published. To publish a tool, press the 'Publish' button in the
+top-right corner.
+
+For the tutorial, generally, you should hit the "Refresh All" button to
+make sure Dockstore has examined your latest repositories on Quay. This
+is how Dockstore picks up new repositories like we did here. (After the
+tutorial, you can refresh tools individually or by organization to speed
+things up)
+
+Now select the ``collaboratory/dockstore-tool-bamstats`` repository and
+click "Publish". The tool is now listed on Dockstore!
+
+You can also click on the "Versions" tab and should notice ``1.25-6`` is
+present and valid. If any versions are invalid it is likely due to a
+path issue to the ``Dockstore.cwl`` or ``Dockerfile``. In BAMStats I used the default value
+of ``Dockstore.cwl`` and ``Dockerfile`` in the root repo directory so
+this was not an issue.
+
+.. figure:: /assets/images/docs/versions_toggle.png
+   :alt: Publish
+
+   Publish
+
+Next, pick a version of your tool that you wish to present to the world
```

Comment:
1. Clicking on the version alone does not set the default version; you need to explicitly set the default version.
2. Note the new UI for this in staging
3. The Launch with section should reflect the current version, not the default version.

Reply:
I can address 3 but not sure what you mean on the other 2. This portion was inherited from existing docs, I didn't touch it.

Reply:
> Next, pick a version of your tool that you wish to present to the world by clicking on the radio selector in the Version column.

For 2, there is no radio selector in 1.12. There is a dropdown in the actions column where you set the default version.

For 1, I missed "the radio selector" portion when I first read it, so you can ignore it.

> This portion was inherited from existing docs, I didn't touch it.

Agreed, but like I said earlier, it showed up as a green diff, so I reviewed it. :)

Reply:
Hope I didn't come off as "you shouldn't have reviewed it," because your reviews of this content have caught a lot of small errors that I missed. Just wasn't familiar the text so I wasn't sure what you meant. But I see what you mean now. I wonder if there's a way we can keep track of specific mentions of the UI in our docs so we can flag such changes in the future?